src/i965_drv_video.h

```diff
@@ -69,7 +69,9 @@
 #define DEFAULT_SATURATION      50
 
 #define ENCODER_QUALITY_RANGE     2
+#define ENCODER_QUALITY_RANGE_AVC     8
```

Comment:
AVC kernel support quality level 1-7 by now. The test result show 1-3 is closer and 4-6 is closer.

MediaSDK support 1-7.